### DISCUSSION TITLE
MDN community call on May 29th

### DISCUSSION THREAD

**Discussion by pepelsbey** [mdn/mdn-community#685] [Announcements]
Hey! The next MDN web docs community call is scheduled next week on Wednesday, May 29th, at 17:00 CET 🗓️

The monthly community call is a place where we discuss everything related to contributing to MDN, sharing news and knowledge, and helping each other to support the web platform. Feel free to send PRs [to the agenda](https://github.com/mdn/community-meetings/blob/main/2024-05-29/agenda.md) if you’d like to share or discuss anything. Or just let us know if you have any ideas!

- [Discord event](https://discord.com/events/1009925603572600863/1242798722749038644)
- [Meeting agenda draft](https://github.com/mdn/community-meetings/blob/main/2024-05-29/agenda.md)

See you there! 👀

**Comment by pransh15** [comment]
Looking forward to it! 🎉
